docs/event_details/screen_opened.md

```diff
@@ -0,0 +1,24 @@
+# screen_opened
+
+## When the event is called
+
+If you’d like to track when screen is opened and when user navigates back to this screen from previous screen.
+
+## Parameters
+
+- event_name (required) - Always “screen_opened”.
+- item_name (required) - Unique name of the screen that is opened.
```

Comment:
**chore**: I made pretty important mistake, when reviewing the PR first time.
It's very impotant for events that are related to some screen, to include screen name where given action was executed. This allows ensuring given event is related to given screen.
So, all events that are related to screen, should have screen name included in it event. 

So button click should have:
```json
{
  "event_name": "button_click",
  "item_name": "Check-in button",
  "screen_name": "reviewers_screen",
}
```

also when screen is opened, we should use the same "screen_name" parameter:
```json
{
  "event_name": "screen_opened",
  "screen_name": "reviewers_screen"
}
```

Sorry that I haven't found out this before because this will cause a lot of changes :/ 

You can use "revieres_screen" or "Reviewers screen" doesn't metter, but it should be consistent.

Keep in mind that for `simple_action`, `action_start` and `action_finished` this `screen_name` is required but only if given action is related to a screen. If it's some background work, this isn't required.